src/main/java/com/rebook/studygroup/service/command/StudyGroupCommand.java

```diff
@@ -0,0 +1,21 @@
+package com.rebook.studygroup.service.command;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@Getter
+public class StudyGroupCommand {
```

Comment:
스터디그룹 생성 커맨드라고 명명해주는게 더 이해하기 쉬울 것 같습니다~! 향후 업데이트 명령 삭제 등과 구분을 위해서 구체적으로 지정해주는게 좋아요.

```suggestion
public class StudyGroupCreateCommand {
```

Reply:
> 스터디그룹 생성 커맨드라고 명명해주는게 더 이해하기 쉬울 것 같습니다~! 향후 업데이트 명령 삭제 등과 구분을 위해서 구체적으로 지정해주는게 좋아요.

지적 감사합니다!